pkg/sql/logictest/testdata/logic_test/alter_primary_key

```diff
@@ -1602,6 +1604,10 @@ COMMENT ON INDEX public.pkey_comment@pkey IS 'idx';
 COMMENT ON CONSTRAINT i2 ON public.pkey_comment IS 'idx3';
 COMMENT ON CONSTRAINT pkey ON public.pkey_comment IS 'const'
 
+# Ensure `SHOW CREATE` does not break (issue #114081).
```

Comment:
nit: Can you include a short description of #114081 ? If someone is reading through this code, having to navigate to a different resource to even get a hint of what's happening can be frustrating.

What would you think about:
```
# Ensure that comment references have been appropriately updated by running `SHOW CREATE` (See issue #114081 for details)
```